docs/JK.Common/Patterns/Specification/AndSpecification_TEntity.md

```
@@ -0,0 +1,29 @@
+[Docs](../../../README.md) > [JK.Common](../../README.md) > AndSpecification<TEntity>
+
+# AndSpecification<TEntity>
+
+**Namespace:** `JK.Common.Patterns.Specification`
+
+Generic And specification.
+
+**Type Parameter:** `TEntity` — The type of the entity.
+
+### #ctor
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)